src/components/ForecastMainBanner.svelte

```diff
@@ -4,6 +4,8 @@
 
   import weatherCodes from '../assets/weather-interpretation-code-description.json';
```

Comment:
It's not in the scope of the task but please add `cursor:pointer` in these 3 places 
![image](https://github.com/user-attachments/assets/46ea41ef-d618-491f-ad97-2bbc2c6e8adb)